be/src/vec/exec/ves_http_scan_node.cpp

```diff
@@ -30,68 +35,228 @@ namespace doris::vectorized {
 
 VEsHttpScanNode::VEsHttpScanNode(ObjectPool* pool, const TPlanNode& tnode,
                                  const DescriptorTbl& descs)
-        : EsHttpScanNode(pool, tnode, descs) {
-    _vectorized = true;
+        : ScanNode(pool, tnode, descs),
+          tuple_id_(tnode.es_scan_node.tuple_id),
+          runtime_state_(nullptr),
+          tuple_desc_(nullptr),
+          num_running_scanners_(0),
+          scan_finished_(false),
+          eos_(false),
+          max_buffered_batches_(1024),
+          wait_scanner_timer_(nullptr) {}
+
+Status VEsHttpScanNode::init(const TPlanNode& tnode, RuntimeState* state) {
```

Comment:
This file has a lot of similar code to 'es_http_scan_node.cpp'.

Reply:
Hi @carlvinhust2012 , thanks for you reviewing. As described in issue #9965 , this PR is just for refactoring. Futhermore, this PR make all scan nodes on vectorized independent from row-based ones so we could remove all row-based codes in the near future. You could just refer to this issue for motivations for this PR.